bebraven/platform
# Add ENV vars to configure the outgoing mailer.

Release Plan:
- Configure the ENV vars in this commit in staging and production upon release.
- Staging should have a MAILER_DELIVERY_OVERRIDE_ADDRESS set. Let's setup a google
  group to act as a sink for outgoing staging emails for now. We could eventually
  switch to something link mailtrap.io

Test Plan:
- Set the ENV vars to a real SMTP server and make sure you can send email and it hits
  your inbox.
- Set the MAILER_DELIVERY_OVERRIDE_ADDRESS env var to your address and try
  send an email to someone else using ActionMailer. It should go to you.
